src/components/TouchableOpacity.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * https://github.com/facebook/react-native/blob/master/Libraries/Components/Touchable/TouchableWithoutFeedback.js
+ */
+import React from 'react';
+import EdgeInsetsPropType from '../ropTypes/EdgeInsetsPropType';
+import View from './View';
+// import Touchable from './Touchable';
```

Comment:
Ah yea I see it now. Personally i'd try and avoid overriding the history like that, avoids any confusion like this, and it's generally a bad idea